pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -54,8 +55,9 @@ python3Packages.buildPythonApplication rec {
       python-magic progressbar33 pypdf2 tlsh
     ])
     ++ lib.optionals stdenv.isLinux [ python3Packages.pyxattr python3Packages.rpm acl cdrkit dtc ]
+    ++ lib.optionals (enableBloat && enableUnfree) [ apktool ]
```

Comment:
does this imply apktool is unfree? according to [the package search](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=apktool) it has [Apache License 2.0](https://spdx.org/licenses/Apache-2.0.html)

Reply:
it pulls an unfree dependency I suppose related to Android

Reply:
Can you add as a comment to the code which one, so it can be understood later.

Reply:
got it

Reply:
build-tools is unfree which comes from ``inherit (androidenv.androidPkgs_9_0) build-tools;`` and for that you need to accept the androidsdk license with an extra attribute within config.